components/toast/stories/toast.stories.js

```diff
@@ -84,6 +109,39 @@ Positive.parameters = {
 	chromatic: { disableSnapshot: true },
 };
 
+/**
+ * They must include text to communicate a message. Write the text as concisely as possible while still being clear about what has happened or is happening.
```

Comment:
It's a little unclear what the subject is here, you might exchange "They" for "Toasts" here.